dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/configurator/parser/model/ConfigItem.java

```diff
@@ -37,6 +38,50 @@
     private Map<String, String> parameters;
     private String side;
 
+    @SuppressWarnings("unchecked")
+    public static ConfigItem parseFromMap(Map<String, Object> map) {
+        ConfigItem configItem = new ConfigItem();
+        configItem.setType((String) map.get("type"));
+
+        Object enabled = map.get("enabled");
+        if (enabled != null) {
+            configItem.setEnabled(Boolean.parseBoolean(enabled.toString()));
+        }
+
+        Object addresses = map.get("addresses");
+        if (addresses != null) {
+            configItem.setAddresses(((List<Object>) addresses).stream()
```

Comment:
it would be better to check if addresses is align from `List<Object>`

Reply:
done